web/src/components/molecules/Common/ResizableProTable/resizble.tsx

```diff
@@ -0,0 +1,36 @@
+import { Resizable } from "react-resizable";
+import type { ResizeCallbackData } from "react-resizable";
+
+export type { ResizeCallbackData } from "react-resizable";
+
+export const ResizableTitle = (
+  props: React.HTMLAttributes<any> & {
+    onResize: (e: React.SyntheticEvent<Element>, data: ResizeCallbackData) => void;
+    width: number;
+    minWidth: number;
```

Comment:
Both necessary?

Reply:
@KaWaite , yes, one is changeable the other is fixed and defines the minimum value for narrowing the table.